### ISSUE TITLE
Loader appears centered on page

### ISSUE THREAD

**Issue by colleenskemp** [ProjectDrawdown/global-research-platform-web-ui#21]
Time Estimate: 

Problem: Currently the loading icon is appearing at the bottom of the page. This looks odd.

Acceptance criteria: 
Loading icon should appear in the center of the page or in the top 1/3 of page.  

Link to associated JIRA ticket: https://colabcoop.atlassian.net/browse/PDD-370

**Comment by brodavi** [comment]
@eethann sorry I didn't understand the comment in Jira, can you rephrase?

**Comment by brodavi** [comment]
okay PR is updated @eethann 

created a LoadingSpinner component to be re-used in those 3 places, and adjusted it to use fixed positioning

**Comment by shrutimehta13** [comment]
resolved on stage
<img width="1384" alt="Screenshot 2021-07-12 at 5 22 39 PM" src="https://user-images.githubusercontent.com/34771314/125284410-0ade9d80-e337-11eb-805f-cfc342e730df.png">
